ephios/extra/templatetags/utils.py

```diff
@@ -38,6 +38,11 @@ def dict_get(dictionary, key):
     return dictionary.get(key)
 
 
+@register.filter
+def get_items(dictionary, key):
+    return dictionary.get(key).items()
```

Comment:
I think the name here is misleading. Either use a better template context, implement a more specific filter/tag or use a better name here.

Reply:
How about `get_items_for_key`?